src/Options.cs

```diff
@@ -0,0 +1,81 @@
+﻿using CommandLine;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DarkRift.Cli
+{
+    [Verb("new", HelpText = "Create a new DarkRift project.")]
+    public class NewOptions
+    {
+        [Option('f', Default = false, HelpText = "Force creation overwriting any files that already exist in the directory.")]
+        public bool Force { get; set; }
+
+        [Value(0, HelpText = "The name of the template to unpack.", Required = true)]
+        public string Type { get; set; }
+
+        [Value(1, HelpText = "The directory to unpack the template in.")]
+        public string TargetDirectory { get; set; }
+
+        [Option("version", HelpText = "Specify the DarkRift version to use.")]
+        public Version Version { get; set; }
+
+        [Option('p', "pro", Default = false, HelpText = "Use the pro version.")]
+        public bool Pro { get; set; }
+
+        [Option('s', "platform", Default = ServerPlatform.Framework, HelpText = "Specify the .NET platform of the server to use.")]
+        public ServerPlatform Platform { get; set; }
+    }
+
+    [Verb("run", HelpText = "Run a DarkRift project.")]
+    public class RunOptions
+    {
+        [Value(0)]
+        public IEnumerable<string> Values { get; set; }
+    }
+
+    [Verb("get", HelpText = "Downloads a plugin package into this server.")]
+    public class GetOptions
+    {
+        [Value(0, Required = true)]
+        public string Url { get; set; }
+    }
+
+    [Verb("pull", HelpText = "Pulls the specified version of DarkRift locally.")]
+    public class PullOptions
+    {
+        [Value(0, Required = false)]
+        public Version Version { get; set; }
+
+        [Option('p', "pro", Default = false, HelpText = "Use the pro version.")]
+        public bool Tier { get; set; }
+
+        [Option('s', "platform", Default = ServerPlatform.Framework, HelpText = "Use the .NET platform of the server to use.")]
+        public ServerPlatform Platform { get; set; }
+
+        [Option('d', "docs", Default = false, HelpText = "Download the documentation for this version instead.")]
+        public bool Docs { get; set; }
+
+        [Option('l', "list", Default = false, HelpText = "List installed versions of DarkRift")]
+        public bool List { get; set; }
+
+        [Option('f', "force", Default = false, HelpText = "Forces downloading of a DarkRift version or documentation")]
+        public bool Force { get; set; }
+
+        [Option("latest", Default = false, HelpText = "Specifies latest version")]
```

Comment:
I personally think version should be streamlined to numbers, its just a lot easier to manage, in the case of RC its easily fixable, but it wouldn't have the same meaning anymore probably. The solution would be the RC would be the actual release version and any changes would go in the Minor part of the version, another solution would be, since those are release RCs they shouldn't be considered releases, so the CLI wouldn't download them. Of course that would defeat the point of the CLI managing Dark Rift version so option one would be the best. Second option would obviously be completely discard version and keep string

Reply:
I think my biggest thing with this change is I don't yet see a reason to do it.

`Version` has two advantages, a fixed format and comparability; as noted, the fixed format doesn't match the format used by DR and we don't yet have a need to compare DR version numbers in this tool.

By changing to version we lose the ability to use `latest` and `dev` as special version numbers (although we could handle those for the most part as special cases) and we lose the ability to do `-beta`/`-rc1` style releases on DR because this tool can't support them.

I just see little tangible gain and a lot of drawbacks to changing this. It seems unecessary at this point.